src/command.ts

```diff
@@ -108,18 +126,23 @@ export default abstract class Command {
 
   async _run<T>(): Promise<T | undefined> {
     let err: Error | undefined
+    let result
     try {
       // remove redirected env var to allow subsessions to run autoupdated client
       delete process.env[this.config.scopedEnvVarKey('REDIRECTED')]
-
       await this.init()
-      return await this.run()
+      result = await this.run()
```

Comment:
The method `toSuccessJson` expects `Record<string, any>`. If we want to keep that, then you need to update line 179 to `abstract run(): PromiseLike<T>` where T is defined on command: `export default abstract class Command<T extends Record<string, any> = unknown>{`

Otherwise, you could update toSuccessJson to accept any like the run method returns now.